### ISSUE TITLE
License

### ISSUE THREAD

**Issue by theoparis** [Sakeran/fem-kanban#2]
Hello there. This looks like a really nice project. I found it [on frontend mentor](https://www.frontendmentor.io/solutions/kanban-app-with-svelte-tailwindcss-storybook-bHyKTEZLMB) while I was searching for `tailwind svelte kanban` on google 😄 .

I was wondering if I could make this into a fully featured project since the UI is already pretty complete (open source of course) but since there is no license I wasn't sure if that was ok with you. I've been wanting to create a task management app for a while now as an alternative to notion or trello.

TLDR: Can I re-use this project under the MIT or MPL-2.0 license?

Thanks.

**Comment by Sakeran** [comment]
Hi Theo,

I've actually been approached about the licensing for this project before. The main thing keeping me from adding an explicit liscence is Frontend Mentor's own copyright to the application's design and branding. Due to how Frontend Mentor works, it's necessary to include some of those assets and styles in the source code for the sake of the challenge. So while I'd like to just put up an MIT license for my code and be done with it, I don't have the expertise and/or patience required to do that in a way that feels safe. Therefore, the repository has no license file.

So far as my own code goes: anyone is welcome to use anything *I* developed for this project as if it were MIT-licensed. This includes most of the underlying app structure, and most of the (unstyled) components, but not things like icons, logos, or colors. More succinctly: this repo is open-source, but Frontend Mentor is not.

With that said, I'm not really qualified to give advice on how to safely split *my* stuff from *their* stuff. For example, I couldn't tell you whether or not the UI layout is copyrightable. Depending on what you want to do, this might pose a problem, and you may need to reach out to FEM yourself if you're unsure about something.

I'm sorry I can't be more clear-cut in this situation. You do have my permission to use anything in this repository, provided I am at liberty to grant it. I wish you the best of luck with your project, though, and I hope you're able to find something that works for you if this doesn't.

Thanks!
